src/angularjs/src/app/places/list/place-list.html

```diff
@@ -10,20 +10,32 @@ <h2 class="sidebar-title">City/Town Search</h2>
         <div class="row">
             <div class="column">
                 <div class="form-group">
-                    <label for="1">Filter by city or town name</label>
-                    <input type="text" class="form-control">
+                    <label for="neighborhood-filter">Filter by city or town name</label>
+                    <ui-select
+                        name="neighborhood-filter"
+                        ng-model="placeList.neighborhoodFilter"
+                        title="Filter by Neighborhood">
+
+                        <ui-select-match allow-clear="true" placeholder="Neighborhood" >
+                            {{$select.selected.label}}
+                        </ui-select-match>
+                        <ui-select-choices repeat="neighborhood.uuid as neighborhood in
```

Comment:
I wonder if a select dropdown in this style will get super unwieldy when there are 300+ neighborhoods in this list? @designmatty should we just revert to a standard dropdown that only shows matching results once the user has started to type?